dev/ci/ci-common.sh

```diff
@@ -12,16 +12,25 @@ ls `pwd`/bin
 mathcomp_CI_BRANCH=master
 mathcomp_CI_GITURL=https://github.com/math-comp/math-comp.git
 
+# Where we clone and build external developments
+CI_BUILD_DIR=`pwd`/_build_ci
+
 # git_checkout branch
```

Comment:
Please add a comment with the intended semantics of the function, something such as:
> `git_checkout branch url dest` will checkout a git repository to branch blah or update it if it exists....

Indeed, the semantics of the fucntion is not yet clear to me.